File: Java/075. Soma2/Soma2.java
import java.util.Scanner;
public class Soma2
{
    public static void main(String[] args)
    {
        Scanner scan = new Scanner(System.in);
        int N  = scan.nextInt();
        long soma = 0;
        for (int i = 1; i <= N; i++)
        {
            int numero = scan.nextInt();
            soma += numero;
        }
        System.out.println(soma);
        scan.close();
    }
}

File: C++/075. Soma2/Soma2.cpp
#include <iostream>
int main()
{
    int N;
    int x;
    int soma = 0;
    std::cin >> N;   
    for (int i = 0; i < N; i++)
    {
        std::cin >> x;
        soma += x;
    }    
    std::cout << soma << std::endl;
    return 0;
}